docs/pages/config-plugins/plugins-and-mods.mdx

```diff
@@ -264,18 +264,16 @@ A mod plugin gets passed a `config` object with additional properties `modResult
 Say you wanted to write a mod to update the Xcode Project's "product name":
 
 ```ts my-config-plugin.ts
-import { ConfigPlugin, withXcodeProject } from 'expo/config-plugins';
+import { ConfigPlugin, withXcodeProject, IOSConfig } from 'expo/config-plugins';
 
-const withCustomProductName: ConfigPlugin = (config, customName) => {
+const withCustomProductName : ConfigPlugin<string> = (config, customName) => {
```

Comment:
```suggestion
const withCustomProductName: ConfigPlugin<string> = (config, customName) => {
```

Default Prettier syntax we use in docs, on formatting, removes this space. I also don't think it is required when adding the type of `withCustomProductName`.